target/linux/qualcommax/files/arch/arm64/boot/dts/qcom/ipq8074-wax630.dts

```diff
@@ -0,0 +1,253 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+/dts-v1/;
+
+#include "ipq8074.dtsi"
+#include "ipq8074-hk-cpu.dtsi"
+#include "ipq8074-ess.dtsi"
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+	model = "Netgear WAX630";
+	compatible = "netgear,wax630", "qcom,ipq8074";
+
+	aliases {
+		serial0 = &blsp1_uart5;
+		
+		ethernet0 = &dp6_syn;
+		ethernet1 = &dp4;
+		label-mac-device = &dp6_syn;
+
+		led-boot = &led_system_blue;
+		led-failsafe = &led_system_red;
+		led-running = &led_system_green;
+		led-upgrade = &led_system_blue;
+	};
+
+	chosen {
+		stdout-path = "serial0:115200n8";
+		bootargs-append = " root=/dev/ubiblock0_1";
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "reset";
+			gpios = <&tlmm 54 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+	};
+
+	led_spi {
+		compatible = "spi-gpio";
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		sck-gpios = <&tlmm 18 GPIO_ACTIVE_HIGH>;
+		mosi-gpios = <&tlmm 19 GPIO_ACTIVE_HIGH>;
+
+		led_gpio: led_gpio@0 {
+			compatible = "fairchild,74hc595";
+			reg = <0>;
+			gpio-controller;
+			#gpio-cells = <2>;
+			registers-number = <1>;
+			enable-gpios = <&tlmm 20 GPIO_ACTIVE_HIGH>;
+			spi-max-frequency = <1000000>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_system_red: system-red {
+			default-state = "off";
+			label = "system:red";
+			gpios = <&tlmm 22 GPIO_ACTIVE_LOW>;
+		};
+
+		led_system_green: system-green {
+			default-state = "off";
+			label = "system:green";
+			gpios = <&tlmm 38 GPIO_ACTIVE_LOW>;
+		};
+
+		led_system_blue: system-blue {
+			default-state = "off";
+			label = "system:blue";
+			gpios = <&tlmm 21 GPIO_ACTIVE_LOW>;
+		};
+
+		led_lan1_g {
```

Comment:
Underscores should not be used as part of node names.

So, please rename this into something like: `lan1-green`

Reply:
Done.